src/m_disort.cc

```diff
@@ -362,12 +362,9 @@ void DisortCalcWithARTSSurface(Workspace& ws,
 
 
   } else {
-    
-    ARTS_USER_ERROR_IF(aa_grid.nelem() != 1,
-                       "There is no star and therefore no azimuth dependency.\n",
-                       "The length of th aa_grid should be 1, but is",
-                       aa_grid.nelem(),
-                       "instead.")
+    CREATE_OUT3;
+    out3 << "There is no star and therefore no azimuth dependency.\n";
+    out3 << "The length of th aa_grid should be 1, but is" << aa_grid.nelem() << "instead.\n";
```

Comment:
I really think this is formulated poorly.  You might want aa_grid to have a size elsewhere.  Simply state in `out3` that 

```cpp
"Disort calculation encountered aa_grid size larger than 1 in a case when it does not use aa_grid.  Calculations are performed as if there is no aa_grid."
```

Reply:
I agree this is better.